src/widgets/dnshole/DnsHoleControls.tsx

```diff
@@ -201,38 +253,51 @@ function DnsHoleControlsWidgetTile({ widget }: DnsHoleControlsWidgetProps) {
                 </Box>
-                <Stack spacing="0rem">
+                <Stack spacing="0rem" align="center">
                   <Text>{app.name}</Text>
-                  <UnstyledButton
-                    onClick={() =>
-                      toggleDns(dnsHole.status === 'enabled' ? 'disable' : 'enable', [app.id])
-                    }
-                    disabled={fetchingDnsSummary || changingStatus}
-                    style={{ pointerEvents: enableControls ? 'auto' : 'none' }}
-                  >
-                    <Badge
-                      variant="dot"
-                      color={dnsLightStatus(fetchingDnsSummary || changingStatus, dnsHole.status)}
-                      styles={(theme) => ({
-                        root: {
-                          '&:hover': {
-                            background:
-                              theme.colorScheme === 'dark'
-                                ? theme.colors.dark[4]
-                                : theme.colors.gray[2],
-                          },
-                          '&:active': {
-                            background:
-                              theme.colorScheme === 'dark'
-                                ? theme.colors.dark[5]
-                                : theme.colors.gray[3],
+                  <Flex direction="row" gap="md">
+                    <UnstyledButton
+                      onClick={() =>
+                        toggleDns(dnsHole.status === 'enabled' ? 'disable' : 'enable', [app.id])
+                      }
+                      disabled={fetchingDnsSummary || changingStatus}
+                      style={{ pointerEvents: enableControls ? 'auto' : 'none' }}
+                    >
+                      <Badge
+                        variant="dot"
+                        color={dnsLightStatus(fetchingDnsSummary || changingStatus, dnsHole.status)}
+                        styles={(theme) => ({
+                          root: {
+                            '&:hover': {
+                              background:
+                                theme.colorScheme === 'dark'
+                                  ? theme.colors.dark[4]
+                                  : theme.colors.gray[2],
+                            },
+                            '&:active': {
+                              background:
+                                theme.colorScheme === 'dark'
+                                  ? theme.colors.dark[5]
+                                  : theme.colors.gray[3],
+                            },
                           },
-                        },
-                      })}
+                        })}
+                      >
+                        {t(dnsHole.status)}
+                      </Badge>
+                    </UnstyledButton>
+                    <ActionIcon
+                      size={20}
+                      radius="xl"
```

Comment:
Add `top="2px"`
The badge and the icon are not aligned correctly by default because of the "pause" symbol x)

Reply:
![image](https://github.com/ajnart/homarr/assets/26098587/11d01c89-c07b-4f02-8503-30de6872c09f)
1px isn't enough x) that's why I said 2
After further investigation, I found out that the badge has a 2.67px marge at the top because of the unstyledButton. So the correct value here should be 2.67px.